run-app/startMongo.bat

```diff
@@ -2,37 +2,39 @@
 @REM
 @REM Example usage:
 @REM
-@REM startMongo.sh <databaseName> <databasePort> [mongoPath] 
+@REM startMongo.sh <databaseName> <databasePort> <buildDir> [mongoPath] 
 @REM
 @REM <databaseName> : The name of the directory to create and use to hold the Mongo DB
 @REM <databasePort> : The port with which to start the Mongo DB server
+@REM <buildDir>     : Name of the build directory (Gradle = build, Maven = target)
 @REM [mongoPath] : Optional absolute path to the Mongo install bin directory
 
 @echo off
 
 @REM Get the needed params.
 set dbName=%1
-
 set dbPort=%2
-set mongoPath=%3
+
+set buildDir=%3
+set mongoPath=%4
 
 @REM Set the mongoDB path.
 IF defined mongoPath SET PATH=%PATH%;%mongoPath%
 
 @REM Create the needed directories.
 if defined dbName (
-  IF NOT exist .\target\%dbName% (
-      md .\target\%dbName%
+  IF NOT exist %buildDir%\%dbName% (
+      md .\%buildDir%\%dbName%
```

Comment:
Why does one line include `.\` before %buildDir%, and the other does not? Before `target` was preceded by `.\` on both lines.